jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/core/assembly/DockerAssemblyManager.java

```diff
@@ -132,6 +132,11 @@ public File createDockerTarArchive(
 
                 verifyGivenDockerfile(dockerFile, buildConfig, params.getProperties(), log);
                 interpolateDockerfile(dockerFile, buildDirs, params.getProperties());
+                if (DockerFileUtil.isSimpleDockerFileMode(params.getBasedir())
+                        && dockerFile.getAbsolutePath().equals(DockerFileUtil.getTopLevelDockerfile(params.getBasedir()).getAbsolutePath())) { // Simple Dockerfile mode
```

Comment:
@manusa : I added this additional check to verify whether `Dockerfile` in root directory actually matches the one present in `ImageConfiguration`. Right now, `docker-file-provided` quickstart seems to be picking correct `Dockerfile`.Shall I remove this check or keep it?